tests/testflows/map_type/configs/clickhouse/common.xml

```diff
@@ -0,0 +1,6 @@
+<yandex>
+    <timezone>Europe/Moscow</timezone>
+    <listen_host replace="replace">::</listen_host>
+    <path>/var/lib/clickhouse/</path>
```

Comment:
It's by default, no need to override it in tests.

Reply:
Removed.